web/gui/dashboard_info.js

```diff
@@ -2828,6 +2834,182 @@ netdataDashboard.context = {
 
     'squidlog.mime_type_requests': {
         info: 'The content type of the object as seen in the HTTP reply header. Please note that ICP exchanges usually don\'t have any content type.'
-    }
+    },
+
+    // ------------------------------------------------------------------------
+    // CockroachDB
+
+    'cockroachdb.process_cpu_time_combined_percentage': {
+        info: 'Current combined cpu utilization, calculated as <code>(user+system)/num of logical cpus</code>.'
+    },
+
+    'cockroachdb.host_disk_bandwidth': {
+        info: 'Summary statistics across all system host disks.'
+    },
+
+    'cockroachdb.host_disk_operations': {
+        info: 'Summary statistics across all system host disks.'
+    },
+
+    'cockroachdb.host_disk_iops_in_progress': {
+        info: 'Summary statistics across all system host disks.'
+    },
+
+    'cockroachdb.host_network_bandwidth': {
+        info: 'Summary statistics across all system host network interfaces.'
+    },
+
+    'cockroachdb.host_network_packets': {
+        info: 'Summary statistics across all system host network interfaces.'
+    },
```

Comment:
There is chart title

https://github.com/netdata/go.d.plugin/blob/1f8bc2ac8b084087efba497528e0a5b6646403fe/modules/cockroachdb/charts.go#L166

> Host Disk Cumulative Bandwidth

This is just additional info.

Reply:
Right, but the text from `info` displays right under the title, right? It just seems odd to me that multiple charts would have the same contextual information. They're different charts, so they should have different info. Otherwise, why put it there at all?

Reply:
but ok, makes sense